samples/Samples.h

```diff
@@ -72,6 +72,12 @@ extern "C" {
 #define MAX_SIGNALING_CLIENT_METRICS_MESSAGE_SIZE 736 // strlen(SIGNALING_CLIENT_METRICS_JSON_TEMPLATE) + 20 * 10
 #define MAX_ICE_AGENT_METRICS_MESSAGE_SIZE        113 // strlen(ICE_AGENT_METRICS_JSON_TEMPLATE) + 20 * 2
 
+#define ADJUSTMENT_INTERVAL_SECONDS 1 * HUNDREDS_OF_NANOS_IN_A_SECOND
+#define MIN_VIDEO_BITRATE_KBPS      512     // Unit kilobits/sec. Value could change based on codec.
+#define MAX_VIDEO_BITRATE_KBPS      2048000 // Unit kilobits/sec. Value could change based on codec.
```

Comment:
max is in Gbps?

Reply:
Essentially, yes. The limit is for gstreamer x264enc element and the unit is kbps. Hence the comment that these values could change based on the codec being used. The limit is necessary to make sure gstreamer does not fail with critical warnings being unable to set a value beyond this.